### DISCUSSION TITLE
Allow editing of Label nodes in the 2D screen.

### DISCUSSION THREAD

**Discussion by kit-solent** [godotengine/godot-proposals#10184] [Editor]
Currently the text in a Label node can only be edited from the inspector. I think it would be useful and slightly more intuative to be able to edit this text from within the centeral viewport (in addition to the inspector). Many other control nodes can be moved, anchored, and otherwise manipulated directly through this view. I would also mean you could edit label nodes even when the middle section is expanded, hiding the inspector and other docs. 
![Screenshot 2024-07-12 130245](https://github.com/user-attachments/assets/5ffd5230-14e6-43b3-8079-b416a4d681c6)

**Comment by Calinou** [comment]
Thanks for the proposal! Consolidating in https://github.com/godotengine/godot-proposals/issues/1542.
